components/Button/button.css

```diff
@@ -0,0 +1,62 @@
+/* Buttons */
+.btn {
+  border: 0;
+  cursor: pointer;
+  font-size: 1.2rem;
+  font-weight: 600;
+  margin: 1rem 0;
+  outline: 0;
+  padding: 2rem 4vw;
+  text-align: center;
+  text-transform: uppercase;
+  white-space: nowrap;
+}
+
+.btn--primary {
+  background: var(--color-primary);
+  color: #fff;
+}
+
+.btn--primary:hover {
+  background: #3a8ffd;
+}
+
+.btn--secondary {
+  background: var(--color-difblue);
+  color: #fff;
+}
+
+.btn--secondary:hover {
+  background: #05cdf0;
+}
+
+.btn--accent {
+  background: var(--color-accent);
+  color: #fff;
```

Comment:
Using #fff code many time you can shift it to CSS variable rest looks fine

Reply:
Thanks done